src/views/Results.vue

```diff
@@ -64,24 +67,21 @@ export default {
     layoutClass: function () {
       return this.showSidebar ? "layout-1q3q" : "";
     },
-    showPagination: function () {
-      return this.hits > 0;
-    },
     showSidebar: function () {
       return this.facets.length ? true : false;
     },
   },
   methods: {
     async searchTimdex() {
       this.status.loading = true;
+      this.searchterm = this.$route.query.q;
```

Comment:
Is there a reason that we retrieve and store the search query twice? We store it in `this.searchterm` here, but a few lines lower we also grab it via `let query = qs.stringify(this.$route.query)`. Is there a reason for not stringifying this version, and using the `.q` portion here, versus the other choices below?

Reply:
the `.q` is pulling in just one parameter from the query string for display of the serachterm. The stringified thing here pulls in all of the current and future parameters (which will include facets). I'm sure we could make one less call to `this.$route.query` but I'm not sure it would be more or less confusing. I'll take a look.

Reply:
@matt-bernhardt we need to grab `query.q` here to allow us to have that set regardless of whether the timdex call succeeds (we use it elsewhere regardless of success). We grab the full query as a convenient way to populate all of the data we'll need for the timdex call.

I'm not sure if you are suggesting a change or just trying to understand the logic here. You are definitely correct that `q` is a subset of the full `query` but as we are using them for different purposes I'm not sure it would be clearer to change this. (Also, this changeset just moved this logic outside of the condition as we needed it regardless of success).

Reply:
I'm happy to just understand the reasons for doing it this way - I noted that this was now coming before the if statement, which makes sense as you say. Maybe I need to do some more digging around this operation myself and build my understanding of what's happening with `query.q` versus `query`.

Long story short - if you've looked at this again, in light of my questions, and still think this is the way it needs to be, then I'm not going to object. If it still doesn't make sense to me later we can have a more focused conversation then.

Reply:
Oh gotcha. `query` is the full url query string. So it includes things like `q=whatever you searched for`, `page=1`, `facet_whatever=cheese`.

`query.q` is just pulling out our search term which we've named `q`.

This confusion is why I've started to move us to internally calling the searched for phrase `searchterm` instead of `query` because `query` encompasses a lot more in a vue route and it was making it hard to keep track of.

Reply:
Yeah, that makes sense.